src/Cli/dotnet/commands/dotnet-nuget/NuGetCommandParser.cs

```diff
@@ -90,5 +91,21 @@ private static Command GetVerifyCommand()
 
             return verifyCommand;
         }
+
+        private static Command GetTrustCommand()
+        {
+            var trustCommand = new Command("trust");
+
+            trustCommand.AddArgument(new Argument<string>()
```

Comment:
I think we it is better if we set argument's arity `new Argument<string>() { Arity = ArgumentArity.ExactlyOne }` the command can parse only one argument. We may have to do another PR into SDK repo after addressing https://github.com/NuGet/Home/issues/10788 issue.

@sfoslund - Please correct if my understanding is incorrect. If we set `Arity = ArgumentArity.ExactlyOne` then the I expect the new `dotnet nuget trust` command to accept only one argument from the list passed to `FromAmong` method.

Reply:
Yes, that's right. It will also error if no arguments are provided.

Reply:
We have case `no argument` is provided. `dotnet nuget trust` defaults to `dotnet nuget trust list` in our code, so it'll break.

Reply:
It's consistent with previous `nuget trusted-signers` defaults to `nuget trusted-signers list` experience.

Reply:
If you want to allow no arguments then you'll want the arity to be ZeroOrOne, then.

Reply:
Ok. Thank you.  As @kartheekp-ms suggested here I'll do another PR after I addressing https://github.com/NuGet/Home/issues/10788 issue.

Reply:
We can add `ZeroOrOne` arity in this PR because my guess is after addressing https://github.com/NuGet/Home/issues/10788 issue you may have to add subcommands to `dotnet nuget trust` and the arity may not be relevant there.

Reply:
Then how I pass list commands for suggestion?

Reply:
https://github.com/dotnet/sdk/pull/18061/files#r645254982

Reply:
Ok. I taken your suggestion.

Reply:
Could you be review again please?